metrics.yaml

```diff
@@ -181,22 +181,15 @@ user:
     expires: never
 
   income:
-    type: labeled_boolean
+    type: quantity
     lifetime: ping
     send_in_pings:
       - demographics
-    labels:
-      - band_0_24999
-      - band_25000_49999
-      - band_50000_74999
-      - band_75000_99999
-      - band_100000_149999
-      - band_ge_150000
     description: |
       The user household's combined annual income during the
       past 12 months.
     bugs:
-      - https://github.com/mozilla-rally/rally-core-addon/issues/545
+      - https://github.com/mozilla-rally/rally-core-addon/issues/621
     data_reviews:
       - https://github.com/mozilla-rally/rally-core-addon/pull/139#issuecomment-736024232
```

Comment:
I am assuming this line will have to be updated with the data review (probably a github comment on this PR) approving this.

Reply:
Yes

Reply:
I recommend just adding it to the list and not changing the bug numbers

Reply:
Finding the earlier bugs for reference and context is why this is a list